fe/fe-core/src/main/java/org/apache/doris/alter/SystemHandler.java

```diff
@@ -314,7 +314,7 @@ private static void checkDecommissionWithReplicaAllocation(List<Backend> decommi
             for (Table table : db.getTables()) {
                 table.readLock();
                 try {
-                    if (!table.needSchedule()) {
+                    if (!table.isManagedTable()) {
```

Comment:
Is this the same semantic as the original impl.?

Reply:
the original has bug,  should check MTMV here too.